benefits/static/css/styles.css

```diff
@@ -178,6 +192,11 @@ footer {
   margin-bottom: 1rem;
   padding: 1rem;
   width: 100%;
+  text-transform: uppercase;
+}
+
+.buttons a.btn.btn-lg.btn-outline-primary {
+  text-transform: inherit;
```

Comment:
If the `.btn-primary` is targeted for uppercase, this rule is unneeded.

Reply:
The reason outline primary stays inherit is because in the mockup the outline buttons do not have an uppercase so here I told the text transform to remain inherit. Do you want all btn outlines to also have text transform of uppercase?